src/modules/bootloader/main.py

```diff
@@ -923,7 +956,11 @@ def run():
             return None
 
     try:
-        prepare_bootloader(fw_type)
+        if (boot_loader == "grub" or efi_boot_loader == "grub") and install_hybrid_grub:
+            prepare_bootloader("bios")
+            prepare_bootloader("efi")
+        else:
+            prepare_bootloader(fw_type)
```

Comment:
I think this would be cleaner if we implemented as part of the bootloader selection logic inside `prepare_bootloader()`

Reply:
Moved.